.github/workflows/deploy-integration-swb-reference-byon.yml

```diff
@@ -0,0 +1,115 @@
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+name: Deploy-and-integration-test-swb-reference-BYON
+on:
+  workflow_call:
+    secrets:
+      aws-dev-region:
+        required: true
+      role-to-assume:
+        required: true
+      aws-dev-account-region:
+        required: true
+      slack_webhook_url:
+        required: true
+      CYPRESS_ADMIN_PASSWORD:
+        required: true
+      CYPRESS_ADMIN_USER:
+        required: true
+      COGNITO_DOMAIN:
+        required: true
+      USER_POOL_ID:
+        required: true
+      CLIENT_ID:
+        required: true
+      CLIENT_SECRET:
+        required: true
+      VPC_ID:
+        required: true
+      ALB_SUBNET_1:
+        required: true
+      ALB_SUBNET_2:
+        required: true
+      ECS_SUBNET_1:
+        required: true
+      ECS_SUBNET_2:
+        required: true
+      HOSTED_ZONE_ID:
+        required: true
+
+jobs:
+# deploy only when files specific to swb package changed
+  deploy-and-integration-test-swb-reference:
+    runs-on: ubuntu-20.04
+    environment: dev
+    permissions:
+      id-token: write
+      contents: read
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Base Action
+        uses: ./.github/actions/baseAction
+
+      - name: Configure AWS Credentials
+        # if: steps.changed-swb-files.outputs.any_changed == 'true'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: ${{ secrets.aws-dev-region }}
+          role-to-assume: ${{ secrets.role-to-assume }}
+          role-session-name: OIDCSessionName
+
+      - name: Enter config settings from GH secrets
+        shell: bash
+        run: |
+          echo "
+          vpcId: ${{ secrets.VPC_ID }}
+          userPoolId: ${{ secrets.USER_POOL_ID }}
+          clientId: ${{ secrets.CLIENT_ID }}
+          clientSecret: ${{ secrets.CLIENT_SECRET }}
+          albSubnetIds: [${{ secrets.ALB_SUBNET_1 }}, ${{ secrets.ALB_SUBNET_2 }}]
+          ecsSubnetIds: [${{ secrets.ECS_SUBNET_1 }}, ${{ secrets.ECS_SUBNET_2 }}]
+          hostedZoneId: ${{ secrets.HOSTED_ZONE_ID }}" >> "../../solutions/swb-reference/src/config/swbv2-private.yaml"
+
+      - name: Deploy SWB-Reference
+        # if: steps.changed-swb-files.outputs.any_changed == 'true'
+        run: |
+          cd solutions/swb-reference
+          node ../../common/scripts/install-run-rushx.js compile
+          STAGE='swbv2-private' node ../../common/scripts/install-run-rushx.js cdk bootstrap ${{ secrets.aws-dev-account-region }} -v
+          STAGE='swbv2-private' node ../../common/scripts/install-run-rushx.js cdk-deploy
+          STAGE='swbv2-private' node ../../common/scripts/install-run-rushx.js run-postDeployment
+
+      - name: Deploy SWB-UI
+        # if: steps.changed-swb-files.outputs.any_changed == 'true'
+        run: |
+          cd solutions/swb-ui/ui
+          STAGE='swbv2-private' node ../../../common/scripts/install-run-rushx.js deploy-ui-and-api
```

Comment:
Have we updated the deploy-ui-and-api command to build and deploy the docker image?

Reply:
The docker image has already been created in this environment, so that step is not needed and it won't block this pipeline. But you're right, the deploy-ui-and-api script hasn't been updated to deploy the image yet.

Reply:
Updated